modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -862,6 +906,127 @@ private void _testGetOrganizationsPageWithFilter() throws Exception {
 		assertContains(organization2, (List<Organization>)page.getItems());
 	}
 
+	private void _testGetOrganizationWithNestedFields() throws Exception {
+		com.liferay.portal.kernel.model.Organization organization =
+			OrganizationTestUtil.addOrganization();
+
+		AccountEntry accountEntry1 = _addAccountEntry();
+
+		organizationResource.postAccountOrganization(
+			accountEntry1.getAccountEntryId(),
+			String.valueOf(organization.getOrganizationId()));
+
+		AccountEntry accountEntry2 = _addAccountEntry();
+
+		organizationResource.postAccountOrganization(
+			accountEntry2.getAccountEntryId(),
+			String.valueOf(organization.getOrganizationId()));
+
```

Comment:
can we avoid this? and then we will check if the accountBriefs contains accountEntry1 but not accountEntry2